flexport/latitude
# LDS-1 Searchable Select Resizing Bug

## Description:
SearchableSelectInput resizes every time it’s focused via keyboard, this fixes it. You can also recreate this issue by clicking the forms in very specific sequence, see below example.

Issue caused by `DeprecatedPopper` affecting parent `flex` layout when being closed and child `DropdownList` is not open. 

## Sandbox server:
https://core-79.sandbox3.eng-dev.flexport.com/

## Jira:
https://flexport.atlassian.net/browse/LDS-1

## Manual Testing Plan:
1. Open Core.
2. Visit 'switches' page. (https://core-79.sandbox3.eng-dev.flexport.com/switches)
3. Click forms in a very specific sequence. Or tab through, observe `SearchableSelectInput` form does not resize.

## Screenshots:
Before:
![switch_old](https://user-images.githubusercontent.com/7979583/67443652-807e6500-f5ba-11e9-8425-93e17e1f479e.gif)

After:
![switch_new](https://user-images.githubusercontent.com/7979583/67443660-883e0980-f5ba-11e9-8268-750994e0d1e7.gif)
